### DISCUSSION TITLE
PDF File

### DISCUSSION THREAD

**Discussion by vincentmohiuddin** [ansys/pyfluent#1942] [Q&A] [answered]
Hi, 

how can I read a PDF file for a combustion with pyfluent? The file is stored in the same folder as the case files but I am getting a `RuntimeError` (Unable to read data file as PDF data file has not been loaded).

Kind regards, 
Vincent

**Comment by mkundu1** [comment, marked as answer]
`pdf_file_name` is an argument for `read_case`: https://fluent.docs.pyansys.com/version/stable/api/solver/_autosummary/settings/read_case.html#read-case

**Reply by vincentmohiuddin** [reply]
Thank you for the quick reply. I should have seen the parameter in the API reference, sorry. Anyway I got it to run now, thanks again!
